### ISSUE TITLE
Deprecate RTF

### ISSUE THREAD

**Issue by hallvors** [zeroclipboard/zeroclipboard#651]
If a script asks ZeroClipboard to write RTF contents to the clipboard, we should emit a deprecation warning. RTF is unlikely to be available as a natively supported clipboard format because of security concerns. 

Browser vendors know that data placed on the clipboard crosses the web -> native apps security boundary, and they want to be pretty cautious to avoid malicious payloads (we're discussing measures like transcoding image data to ensure it is safe).

With that in mind, allowing script-generated chunks of RTF to be written to the clipboard seems like a serious risk. Many applications with RTF support are probably old and not as hardened and tested as other stuff. ZeroClipboard needs to start warning users that this functionality is going away.

**Comment by hallvors** [comment]
(It would be nice if this landed before #171)

**Comment by JamesMGreene** [comment]
Good idea, since it sounds like the native HTML Clipboard will not support RTF.  Thanks, @hallvors! :+1:
